wiki/Contests/Skinning_Contest/1/en.md

```diff
@@ -0,0 +1,139 @@
+| ![][flag_RU] [cyperdark](https://osu.ppy.sh/users/9893708) | [WhiteCat 2.1](https://osu.ppy.sh/community/forums/topics/1279815) |
+| ![][flag_NZ] [downy](https://osu.ppy.sh/users/4585661) | [New Zealand](https://osu.ppy.sh/community/forums/topics/1292880) |
+| ![][flag_PL] [h3oCharles](https://osu.ppy.sh/users/8712289) | [Untitled Red](https://osu.ppy.sh/community/forums/topics/1278791) |
+| ![][flag_PH] [Inflique](https://osu.ppy.sh/users/13865585) | [Ai Hayasaka non-weeb ver.](https://osu.ppy.sh/community/forums/topics/1293078) |
+| ![][flag_VE] [Kim Minnie](https://osu.ppy.sh/users/3858685) | [Elation](https://osu.ppy.sh/community/forums/topics/1293200) |
+| ![][flag_VN] [koomo](https://osu.ppy.sh/users/2168518) | [Loona](https://osu.ppy.sh/community/forums/topics/1285299) |
+| ![][flag_EE] [Krizto](https://osu.ppy.sh/users/13474556) | [Minimalist 2.0](https://osu.ppy.sh/community/forums/topics/1286223) |
+| ![][flag_CN] [McEndu](https://osu.ppy.sh/users/14059135) | [Rhombus](https://osu.ppy.sh/community/forums/topics/1288510) |
+| ![][flag_NL] [Mr HeliX](https://osu.ppy.sh/users/2330619) | [bingV](https://osu.ppy.sh/community/forums/topics/1293209) |
+| ![][flag_CA] [Nashboss100](https://osu.ppy.sh/users/15802424) | [The Nash Skin](https://osu.ppy.sh/community/forums/topics/1278022) |
+| ![][flag_ID] [Nikano](https://osu.ppy.sh/users/8196177) | [Neynda.](https://osu.ppy.sh/community/forums/topics/1291076) |
+| ![][flag_RU] [Nikstrim](https://osu.ppy.sh/users/10577559) | [notMiku](https://osu.ppy.sh/community/forums/topics/1292954) |
+| ![][flag_MX] [Nochito](https://osu.ppy.sh/users/15255804) | [Nochito Chocomint](https://osu.ppy.sh/community/forums/topics/1281445) |
+| ![][flag_TR] [NotConsistent](https://osu.ppy.sh/users/13845312) | [Space \[Universal Apocalypse\]](https://osu.ppy.sh/community/forums/topics/1267043) |
+| ![][flag_UA] [Pirasto](https://osu.ppy.sh/users/6766278) | [Prinz Heinrich](https://osu.ppy.sh/community/forums/topics/1292847) |
+| ![][flag_AU] [spiefie99](https://osu.ppy.sh/users/10934017) | [Glassmorphism](https://osu.ppy.sh/community/forums/topics/1269582) |
+| ![][flag_VE] [Suminoze](https://osu.ppy.sh/users/5308780) | [1stTry](https://osu.ppy.sh/community/forums/topics/1285565) |
+| ![][flag_PL] [tearek\_](https://osu.ppy.sh/users/20767121) | [Multicolor Diamond HUD](https://osu.ppy.sh/community/forums/topics/1272664) |
+| ![][flag_VN] [TKieen](https://osu.ppy.sh/users/12561202) | [Randoline](https://osu.ppy.sh/community/forums/topics/1291235) |
+| ![][flag_MX] [xet](https://osu.ppy.sh/users/16274977) | [Who Tao](https://osu.ppy.sh/community/forums/topics/1281532) |
+| ![][flag_PH] [Xyber Voltage](https://osu.ppy.sh/users/10443089) | [FreeMod is EZ](https://osu.ppy.sh/community/forums/topics/1269099) |
+| ![][flag_MX] [ZixkyST](https://osu.ppy.sh/users/11844975) | [ZixStar](https://osu.ppy.sh/community/forums/topics/1278132) |
+
+## Podium
```

Comment:
Should maybe link the winners as there is no easy way to get to them, as their names are on the image only currently.

Maybe would also be nice to show the 3 badges in this section as well?